<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraScheduleWidget</class>
 <widget class="QWidget" name="CameraScheduleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>893</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1103</width>
        <height>891</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,0,0,0,0,1">
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="topButtonsLayout">
         <item>
          <widget class="QCheckBox" name="enableRecordingCheckBox">
           <property name="text">
            <string>Recording</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="licensesButton">
           <property name="text">
            <string>Activate License...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QnLicensesUsageWidget" name="licensesUsageWidget" native="true"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="recordingScheduleLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="scheduleGridGroupBox">
           <property name="title">
            <string notr="true">&lt;Recording Schedule&gt;</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="scheduleGridLayout">
            <item>
             <widget class="QnScheduleGridWidget" name="gridWidget" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>250</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="nx::client::desktop::ScheduleSettingsWidget" name="scheduleSettingsWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottomParametersLayout" stretch="1,1,1">
         <property name="spacing">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="nx::client::desktop::ArchiveLengthWidget" name="archiveLengthWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="motionGroupBox">
           <property name="title">
            <string>Motion Recording</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Pre-Recording</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>recordBeforeSpinBox</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="recordBeforeSpinBox">
                 <property name="suffix">
                  <string notr="true" extracomment="Short for &quot;seconds&quot;">&lt; s&gt;</string>
                 </property>
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Post-Recording</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buddy">
                  <cstring>recordAfterSpinBox</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="recordAfterSpinBox">
                 <property name="suffix">
                  <string notr="true" extracomment="Short for &quot;seconds&quot;">&lt; s&gt;</string>
                 </property>
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="copyScheduleGroupBox">
           <property name="title">
            <string>Copying Schedule</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
            <property name="spacing">
             <number>16</number>
            </property>
            <item>
             <widget class="QPushButton" name="exportScheduleButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Copy Schedule to...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exportWarningLabel">
              <property name="text">
               <string>Apply changes before copying schedule.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnScheduleGridWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/schedule_grid_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnLicensesUsageWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/licensing/licenses_usage_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ArchiveLengthWidget</class>
   <extends>QWidget</extends>
   <header>archive_length_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ScheduleSettingsWidget</class>
   <extends>QWidget</extends>
   <header>schedule_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enableRecordingCheckBox</tabstop>
  <tabstop>licensesButton</tabstop>
  <tabstop>recordBeforeSpinBox</tabstop>
  <tabstop>recordAfterSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>doNotChangeStateChanged(int)</slot>
 </slots>
</ui>
